packages/app/src/components/FormFieldItems.tsx

```diff
@@ -0,0 +1,73 @@
+import {
+  Avatar,
+  HookedValidationError,
+  InputCheckboxGroup,
+  ListItem,
+  Text
+} from '@commercelayer/app-elements'
+import type { InputCheckboxGroupProps } from '@commercelayer/app-elements/dist/ui/forms/InputCheckboxGroup'
+import type { LineItem } from '@commercelayer/sdk'
+import { useMemo } from 'react'
+import { Controller } from 'react-hook-form'
+interface Props {
+  lineItems: LineItem[]
+}
+
+export function FormFieldItems({ lineItems }: Props): JSX.Element {
+  const options: InputCheckboxGroupProps['options'] = useMemo(
+    () =>
+      lineItems.map((item) => ({
+        value: item.id,
+        content: (
+          <ListItem
+            alignIcon='center'
+            alignItems='center'
+            borderStyle='none'
+            icon={
+              item.image_url != null ? (
+                <Avatar
+                  alt={item.name ?? ''}
+                  size='small'
+                  src={item.image_url as `https://${string}`}
+                />
+              ) : undefined
+            }
+            padding='none'
+            tag='div'
+          >
+            <div>
+              <Text size='regular' tag='div' weight='bold'>
+                {item.name}
+              </Text>
+            </div>
+          </ListItem>
+        ),
+        quantity: {
+          min: 1,
+          max: item.quantity
+        }
+      })),
+    [lineItems]
+  )
+
+  if (options.length === 0) {
+    return <div>No items</div>
+  }
+
+  return (
+    <>
+      <Controller
+        name='items'
+        render={({ field: { onChange, value } }) => (
+          <InputCheckboxGroup
+            title='Items'
+            defaultValues={value}
+            options={options}
+            onChange={onChange}
+          />
+        )}
+      />
+      <HookedValidationError name='items' />
+    </>
```

Comment:
We now have `<HookedCheckboxGroup>` to use. It's basically the same code.

Reply:
Alright! Replaced!